solutions/swb-reference/integration-tests/tests/multiStep/dataset.test.ts

```diff
@@ -55,7 +56,7 @@ describe('multiStep dataset integration test', () => {
     const { data: envDetails } = await adminSession.resources.environments.environment(env.id).get();
     const awsRegion = settings.get('awsRegion');
     const mainAccountId = settings.get('mainAccountId');
-    const accessPointName = `${dataSet.id.slice(0, 13)}-mounted-on-${env.id.slice(0, 13)}`;
+    const accessPointName = `${dataSet.id.slice(0, 13)}-mounted-on-${env.id.slice(0, 12)}`;
```

Comment:
its not obvious why we have 13 and 12 here, could we name these to give them some context?

Reply:
This is what a typical `accessPointName` looks like: `dataset-74a53-mounted-on-env-94bceb8c`

`${env.id.slice(0, 12)}` would equal `env-94bceb8c`
`${env.id.slice(0, 13)}` would equal `env-94bceb8c-`